Makefile

```diff
@@ -90,6 +93,9 @@ ifneq ($(wildcard /.dockerenv), )
 	go test -check.v -check.timeout=$(TEST_TIMEOUT) timeout=$(TEST_SUITE_TIMEOUT) $(TESTFLAGS) github.com/docker/containerd/integration-test
 endif
 
+bench: all validate install bundles-rootfs
```

Comment:
Oh, I see what you mean regarding the shim.

maybe use `shim` instead of `all` then